src/main/java/org/sagebionetworks/bridge/user/TestUserHelper.java

```diff
@@ -158,12 +160,14 @@ public void setClientInfo(ClientInfo clientInfo) {
      */
     public static TestUser getSignedInAdmin() {
         if (cachedAdmin == null) {
-            cachedAdmin = getSignedInUser(CONFIG.getAdminSignIn());
+            SignIn signIn = API_SIGN_IN
+                    .email(CONFIG.get(ADMIN_EMAIL_PROPERTY))
+                    .password(CONFIG.get(ADMIN_PASSWORD_PROPERTY));
```

Comment:
Minor: This modifies API_SIGN_IN. This won't break anything, since API_SIGN_IN is otherwise only used for changeApp(), which ignores email and password. But from a code hygeine perspective, it would probably be better to create a new SignIn.

Reply:
Moved to a constant that we don't modify.